content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,569 @@
+# Running Experiments
+
+## Motivation
+
+Machine Learning and Data Science projects usually involve experimentation.
+Motivation of these experiments can range from finding good hyperparameters to
+checking data and concept drift. DVC 2 introduced a new set of commands to
+manage experiments with minimum boilerplate. It allows to run experiments
+defined by pipelines, track their associated data and model files, set
+parameters for each, push experiment parameters and code to Git remotes without
+committing them, create branches and persist them.
+
+In this part of the DVC User's Guide we will explain how DVC runs the
+experiments, setting parameters, running the experiments in parallel, or in
+temporary directories.
+
+If this is the first time you're introduced to Machine Learning experimentation,
+it may be better to start with the
+[Experiment's Trail](/doc/start/experiments/).
```

Comment:
All of this is already in the index (**Experiment Management** main page), probably no need here.

Reply:
I think to incorporate this text into the index page.

Reply:
If your edits to the index are done, I'm moving this. @jorgeorpinel

Reply:
May take a while since #2654 is the branch of a series of nested PRs. Maybe you can stash this for now or make a separate branch (I can incorporate it when ready)? Thanks

Reply:
p.s. if you could review #2719 that would help BTW!

Reply:
OK summarized in af9469ff